nansat/vrt.py

```diff
@@ -1235,6 +1235,11 @@ def get_shifted_vrt(self, shift_degree):
         # Copy self into self.vrt
         shift_vrt = self.get_super_vrt()
 
+        # cancel shift if not longlat projection
+        projection = self.get_projection()
+        if projection[1] != 'dataset' or 'AUTHORITY["EPSG","4326"]]' not in projection[0]:
+            return shift_vrt
```

Comment:
this is tested right? Seems to be from the coveralls report but the total coverage has decreased.

Reply:
This one is tested. But some older lines were not. I've changed the tests to cover some other lines as well.

Reply:
Great :)